cmd/gator/test/test.go

```diff
@@ -72,8 +73,15 @@ func runE(cmd *cobra.Command, args []string) error {
 	// os-independent.
 	fileSystem := getFS(path)
 
+	// fs.FS does not allow the drive prefix for absolute Windows paths.
+	if runtime.GOOS == "windows" {
+		path = path[2:]
+		path = filepath.ToSlash(path)
+	}
+
 	recursive := false
-	if strings.HasSuffix(path, "/...") {
+	// filepath.Abs strips "/..." from the end of Windows paths, so check the original string.
+	if strings.HasSuffix(args[0], "/...") {
```

Comment:
nit: should we use a var here instead of calling args[0] directly in case we need to change the args in the future?

Reply:
Good idea

Reply:
Done.